### ISSUE TITLE
Publisher: Creating context report page

### ISSUE THREAD

**Issue by jakubjezek001** [ynput/ayon-core#1011]
### Is there an existing issue for this?

- [x] I have searched the existing issues.

### Please describe the feature you have in mind and explain what the current shortcomings are?

We can now view the Publishing report tab, which includes logs for each plugin. This feature is useful for exchanging failed publishes via JSON files. We receive these reports from clients daily, but they often lack Creator context, which is crucial as it significantly impacts the publishing process.

### How would you imagine the implementation of the feature?

It would be great to have a Creator report log, similar to the Publish log report.

### Are there any labels you wish to add?

- [x] I have added the relevant labels to the enhancement request.

**Comment by jakubjezek001** [comment]
Yes and it should also be somehow implemented into `publish-report-viewer` ui.

**Comment by iLLiCiTiT** [comment]
Like it. Question is what do we want to collect.

I guess discovered and crashed plugins, we already do collect crashed, but maybe it would make sense to split them.

All created instances, with all attribute definitions? Might be a lot of data (...sensitive data).

How do we want to collect logs and errors... We probably can't "categorize" them like collect/create/remove etc., because the methods are usually shared. They're also not usually "instance specific", so it's just set of all logs and errors?

If we do collect the information then the UX is another question. Do we want to be able see that reports in Publisher too, or just in publish report viewer tool?

**Comment by BigRoy** [comment]
Just linking another issue, because I feel this overlaps with: https://github.com/ynput/ayon-core/issues/819
